CMakeLists.txt

```diff
@@ -20,7 +20,8 @@ if (WIN32)
     message("WIN32 build. Creating: ${CMAKE_BINARY_DIR}/cmake_install/ziti-sdk-${PROJECT_VERSION}")
     file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/cmake_install/ziti-sdk-${PROJECT_VERSION})
     set(CMAKE_INSTALL_PREFIX ${CMAKE_BINARY_DIR}/cmake_install/ziti-sdk-${PROJECT_VERSION})
-    add_subdirectory(windows)
+    # TODO 
+    # add_subdirectory(windows)
```

Comment:
perhaps make an issue for this?

Reply:
I added general issue for build a shared lib already #24